jax/experimental/jax2tf/examples/serving/model_server_request.py

```diff
@@ -15,7 +15,7 @@
 
 See README.md for instructions.
 """
-import grpc
+import grpc  # type: ignore[import]
```

Comment:
For my own edification, could you explain why this is the right approach here?

Reply:
We run `mypy` in OSS, and it complains if it cannot find one of the modules. `grpc` is not a standard requirement for JAX, and we don't want it to be. You need it only to run this serving example. For myself, when `mypy` complains, if I do not see the type error, I just `type: ignore`. My thinking is that we want the type annotations for documentation first, and the actual type checking is best effort.